can/io/logger.py

```diff
@@ -51,17 +62,18 @@ def __new__(
         if filename is None:
             return Printer(*args, **kwargs)
 
-        lookup = {
-            ".asc": ASCWriter,
-            ".blf": BLFWriter,
-            ".csv": CSVWriter,
-            ".db": SqliteWriter,
-            ".log": CanutilsLogWriter,
-            ".txt": Printer,
-        }
+        if not Logger.fetched_plugins:
+            Logger.message_writers.update(
+                {
+                    writer.name: writer.load()
```

Comment:
@felixdivo @sou1hacker 
This is quite old, but is this `load` method actually documented anywhere?